### PR TITLE
Gemfile.lock fix for docker environment.

### PR DESCRIPTION
**Pull request by ElijahCaine**
fixes issue #53

## Changes in this PR.

- [X]  Removes `Gemfile.lock` in `cleanup.sh`.
- [X] Documents above

## Testing this PR.

1. Run `docker-compose run dev bash
2. Run `rake spec`
3. No errors yay!

@osuosl/devs

### CODE REVIEW

File: dockerfiles/cleanup.sh [hunk 1]
```diff
@@ -1 +1,2 @@
 chown --reference=/data/code -R .
+rm -rf /data/code/Gemfile.lock
```
Comment (Kennric) [inline dockerfiles/cleanup.sh]: Surely we don't need -rf here? Gemfile.lock should never be a directory or in need of forcing...

Comment (ElijahCaine) [inline dockerfiles/cleanup.sh]: Oops, force of habit.

**Comment by Kennric** [comment]
+1

**Comment by matthewrsj** [comment]
+1

**Pull request merged**